source/projects/private-voice-assistant/voice_remote_classic_phone.markdown

```diff
@@ -0,0 +1,154 @@
+---
+title: "Setting up an analog phone to control Home Assistant"
+---
+
+This tutorial shows how to set up an analog phone to control Home Assistant with your voice.
+The phone is used to implement a push-to-talk mechanism. 
+In this way, the beginning and end of a voice stream are clearly defined.
+This reduces the complexity of voice processing. 
+
+It also means that Home Assistant only listens when you want it to.
+
+<lite-youtube videoid="bPDyDFUHDCk" videotitle="Using an analog phone to control Home Assistant"></lite-youtube>
+
+## Required material
+
+* Latest version of Home Assistant installed on your Home Assistant device
+* [Analog phone](#about-the-analog-phone)
+* Analog telephone adapter 
+  [Grandstream HT801](https://www.grandstream.com/products/gateways-and-atas/analog-telephone-adaptors/product/ht801) or 
+  [HT802](https://www.grandstream.com/products/gateways-and-atas/analog-telephone-adaptors/product/ht802)
+  * includes a 5&nbsp;V power adapter and an Ethernet cable
+* RJ11 cable to connect the phone to the Grandstream
+
+## Setting up Grandstream
+
+1. Connect the RJ11 cable from the phone to the green socket on the Grandstream.
+   ![Grandstream HT801 ports](/images/assist/grandstream-ht801-interfaces.png)
+1. Connect the Grandstream to your network: 
+   * Plug the Ethernet cable into the blue socket and connect it to your router.
+1. Start up the Grandstream.
+   * Connect the power supply.
+   * Once the Grandstream has booted, the two LEDs for power and Ethernet light up solid blue. The phone LED won't light up.
+1. Log onto your router and identify the IP address of the Grandstream.
+1. Enter the IP address into a browser window and log onto the Grandstream *Device Configuration* software.
+   * The default credentials are:
+     * **Username**: `admin`
+     * **Password**: `admin`  
+   ![Login to Grandstream](/images/assist/grandstream_login.png)
+1. Open the **FXS Port** tab and in the **Offhook Auto-Dial** field, enter the Home Assistant IP address in the following format:
+   * \*47192\*168\*1\*100\*5060
+   ![Define IP for autodial](/images/assist/grandstream_autodial.png)
+   * Note: instead of 192\*168\*1\*100\*, enter the IP address of your Home Assistant instance.
+   * If you have a Grandstream HT802, there are two ports to configure. Repeat this step for the second port.
```

Comment:
The second port can be configured to call a different Home Assistant device, but it might be worth noting that a single Home Assistant device can't distinguish between the two ports.

Reply:
I think that we should remove all mention of the HT802. It makes the tutorial more complicated than necessary

Reply:
Removed the references to the HT802